sdk/transaction/src/wasm.rs

```diff
@@ -0,0 +1,58 @@
+//! `Transaction` Javascript interface
+#![cfg(target_arch = "wasm32")]
+#![allow(non_snake_case)]
+use {
+    crate::Transaction,
+    solana_hash::Hash,
+    solana_keypair::Keypair,
+    solana_program::{message::Message, wasm::instructions::Instructions},
```

Comment:
Since `instructions` and `message` got moved, this shouldn't need solana_program anymore